app/views/browse/_way.html.erb

```diff
@@ -24,17 +24,18 @@
       <h4><%= t ".nodes" %></h4>
       <details <%= "open" if way.way_nodes.count < 10 %>>
         <summary><%= t ".nodes_count", :count => way.way_nodes.count %></summary>
-        <ul class="list-unstyled">
+        <ul class="list-unstyled browse-element-list">
           <% way.way_nodes.each do |wn| %>
-            <li>
+            <%= element_list_item "node", wn.node do %>
               <%= element_single_current_link "node", wn.node %>
               <% related_ways = wn.node.ways.reject { |w| w.id == wn.way_id } %>
+              <% icon_connector = " " %>
```

Comment:
Why use a variable for this when it doesn't ever seem to change?

Reply:
That was the easiest way to stop linters from complaining. Otherwise they tell to use interpolation. But as soon as I use interpolation, html gets escaped.

Reply:
And it won't exist if we do #5317. Not having to deal with inline icons is one of the goals of that PR.